src/neomaril_codex/base.py

```diff
@@ -530,3 +567,7 @@ def download_result(
             raise ExecutionError("Execution failed")
         else:
             logger.info(f"Execution not ready. Status is {self.status}")
+
+    def execution_info(self) -> None:
+        """Show the execution data in a better format"""
+        print(parse_json_to_yaml(self.execution_data))
```

Comment:
Why don't we use the logger instead of the print here?

Reply:
I though about that, but before I create this method, in the codex sample the variable `execution_data` were accessed directly, printing a json, so I tried to keep this format. Change to a log would be better option?

Reply:
Yes, I think it makes sense to have a more consistent code regarding this information presentation level.

Reply:
https://github.com/datarisk-io/mlops-neomaril-codex/pull/59/commits/a3a361ad52668aca74056c5581a3c9c0d3891981